### ISSUE TITLE
"reference_only" has suddenly stopped working

### ISSUE THREAD

**Issue by owlwc** [Mikubill/sd-webui-controlnet#2935]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

Today, when using 'reference_only' to generate images, I found that it suddenly stopped working. 

This is the image information generated without enabling 'reference_only':

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/104346667/46438b61-d327-4317-9542-d50454364ed5)

Here is the image information generated with 'reference_only', however, it did not work.":

![1716788552215](https://github.com/Mikubill/sd-webui-controlnet/assets/104346667/d27ccfca-71dd-4695-8e57-cfb3ae6a6c5c)

But using 'reference_adain' works as expected: 

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/104346667/af8be84b-d4a4-4458-a442-f05576041402)

Below is the log output, and there are no errors reported:

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/104346667/22549af9-86de-4ea6-8b1a-9c9e694b2b03)

This is not normal.

Did I inadvertently change some settings?

### Steps to reproduce the problem

enabled and disabled the reference

### What should have happened?

At least the output generated before enabling 'reference_only' should be different from the output generated after enabling it.

### Commit where the problem happens

webui: v1.9.3
controlnet: v1.1.449 

### What browsers do you use to access the UI ?

Microsoft Edge

### Command Line Arguments

```Shell
Launching Web UI with arguments: --theme light --xformers --no-half-vae --api --autolaunch
```

### List of enabled extensions

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/104346667/80efb91b-9d48-4173-8f11-e8c895e5531d)

### Console logs

```Shell
reference_only:
==================================================================================
2024-05-27 13:41:34,440 - ControlNet - INFO - unit_separate = False, style_align = False
2024-05-27 13:41:34,444 - ControlNet - INFO - Using preprocessor: reference_only
2024-05-27 13:41:34,444 - ControlNet - INFO - preprocessor resolution = 640
2024-05-27 13:41:34,492 - ControlNet - INFO - ControlNet Hooked - Time = 0.05523490905761719
2024-05-27 13:41:34,877 - ControlNet - INFO - ControlNet used torch.float32 VAE to encode torch.Size([1, 4, 80, 48]).
==================================================================================
reference_adain:
==================================================================================
2024-05-27 13:45:16,414 - ControlNet - INFO - Preview Resolution = 640
2024-05-27 13:45:22,231 - ControlNet - INFO - unit_separate = False, style_align = False
2024-05-27 13:45:22,234 - ControlNet - INFO - Using preprocessor: reference_adain
2024-05-27 13:45:22,234 - ControlNet - INFO - preprocessor resolution = 640
==================================================================================
```

**Comment by huchenlei** [comment]
I cannot reproduce this locally. `reference_only` seems to work fine.

**Comment by owlwc** [comment]
> I cannot reproduce this locally. seems to work fine. Can you share detailed reproduction steps from launching webui? It might be due to some bugs, reference_only always applied even you disable the unit.`reference_only`

Thank you for the tip, I've resolved the issue. It seems to have been caused by a third-party launcher. After launching it directly in the command prompt's virtual environment with 'python webui.py', 'reference_only' is working properly now.

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/104346667/e1e8676d-d008-41c6-9066-4727aa8008a2)
